celer/cython_utils.pyx

```diff
@@ -172,13 +174,16 @@ cdef floating dual_lasso(int n_samples, floating alpha, floating norm_y2,
     """Theta must be feasible"""
     cdef int i
     cdef floating d_obj = 0.
+    alpha = 1.
```

Comment:
do it cleanly: remove this line, and implement the formula without the scaling by alpha